### PR TITLE
Update readme with build instructions

### PR DESCRIPTION
**Pull request by anujdeshpande**
- added `j4` flag to`make` command to parallelise builds

### CODE REVIEW

**Review by Bucknalla — APPROVED** [review]
This looks good but I think we should probably take out all of those other devices that dont relate to the fin? What are your thoughts @anujdeshpande ?

File: makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ bgm13:
 	cd builds; rm -rf $@; mkdir $@; cd $@; cmake -DDEVICE=BGM13P22F512GA ../..; make;
 
 bgm1:
-	cd builds; rm -rf $@; mkdir $@; cd $@; cmake -DDEVICE=BGM111A256V2 -DBOARD=BGM11_BRD4300A ../..; make;
+	cd builds; rm -rf $@; mkdir $@; cd $@; cmake -DDEVICE=BGM111A256V2 -DBOARD=BGM11_BRD4300A ../..; make -j4;
```
Comment (Bucknalla) [inline makefile:76]: Now that I think about this maybe `make -j$(nproc)` makes more sense? Maybe we could look at the OS supported Darwin or Linux and offer the maximum number of cores available, i.e. `'sysctl -n hw.ncpu` for mac?

**Review by Bucknalla — CHANGES_REQUESTED** [review]

File: makefile (hunk 1 above)
Comment (Bucknalla) [reply makefile:76]: Mainly because this looks like it might cause issues for anyone compiling on just 2 cores

Comment (anujdeshpande) [reply makefile:76]: Looks like this is the first OS specific check we'll be adding in that case. Should that be a problem?

Comment (Bucknalla) [reply makefile:76]: @anujdeshpande right now this build system only support linux and MacOS (as far as I am aware). I don't think it's an issue to do a quick check before running on available cores.

**Pull request closed without merging**